tests/Makefile

```diff
@@ -72,17 +73,17 @@ clean:
 ifeq (, $(NOT_ALL_TESTS))
 
 test-%:
-	$(CHARON_CMD) --dest $(DEST)/llbc
-#	$(CHARON_CMD) --dest $(DEST)/ullbc --ullbc
+	$(CHARON_CMD) --dest-file $(DEST)/llbc/$*.llbc
+#	$(CHARON_CMD) --dest-file $(DEST)/ullbc/$*.ullbc --ullbc
 # TODO: this fails for now (it seems some closures are turned into globals,
 # and I don't know how to handle them).
-#	$(CHARON_CMD) --dest $(DEST)/llbc_prom --mir_promoted
+#	$(CHARON_CMD) --dest-file $(DEST)/llbc_prom/$*.llbc --mir_promoted
 # TODO: this fails for now (there is some very low-level desugaring happening)
-#	$(CHARON_CMD) --dest $(DEST)/llbc_opt --mir_optimized
+#	$(CHARON_CMD) --dest-file $(DEST)/llbc_opt/$*.llbc --mir_optimized
 
 else
 
 test-%:
-	$(CHARON_CMD) --dest $(DEST)/llbc
+	$(CHARON_CMD) --dest-file $(DEST)/llbc/$*.llbc
```

Comment:
If we keep a `--dest`/`--dest-dir` option then there is no need to modify those rules